src/formulae/formula.repository.ts

```
@@ -56,16 +62,31 @@ export class FormulaRepository {
     public readDefaultFormula(name: string, conditions: ConfigConditionInput[]): FormulaDefault {
         const object: DefaultConfigObject = this.calculationDefaultObjects.find(value => value.name === name)
         if (!object) {
-            const errorMsg = `No [${CALCULATION_DEFAULT}] config object available for system feature [${name}]`
-            throw new UserInputError(errorMsg)
+            throw new HttpException(
+                {
+                    message: ERROR_CONFIG_NOT_FOUND_I18N_KEY,
```

I think we should not tell the user that we couldn't fetch the formula config. In my opinion, it's better to return a 500 error, and we log the error message out. In this way, the user knows that his side's input is OK, but we have a problem in the application. What do you think?